function-support.html

```diff
@@ -1195,7 +1195,7 @@ <h2 id="logic">Logic and Set Theory</h2>
 
 <h2 id="macros">Macros</h2>
 	<div class="indented">
-	<p>Before macros can be used, they must be defined in the KaTeX <a href="https://github.com/Khan/KaTeX#rendering-options">rendering options</a>. Available functions include:</p>
+	<p>Macros can be defined either via  a KaTeX <a href="https://github.com/Khan/KaTeX#rendering-options">rendering options</a> or by <code>\gdef</code>. Available functions include:</p>

```

Comment:
"a KaTeX rendering options" needs to be "KaTeX rendering options" or "a KaTeX rendering option".